lib/json_schema/validator.rb

```diff
@@ -249,6 +249,12 @@ def validate_items(schema, data, errors, path)
           false
         else
           valid = true
+          if data.size > schema.items.count && schema.additional_items.is_a?(Schema)
+            (schema.items.count..data.count-1).each do |i|
```

Comment:
Kind of a nit, but would you mind using `data.count-1` (i.e. spaces around the minus) just to conform with the existing convention here? Thanks!